docs/sources/write/links/_index.md

```diff
@@ -35,7 +35,27 @@ Although these other types of links still function, replace them with one of the
 
 ## Link from source content that's reused as multiple pages
 
-Use `ref` URIs.
+Use a full URL and ensure the mount target has `mount_links: true` in its front matter.
+The `mount_links` front matter tells Hugo to link to mounted content in the current project if it exists.
+
+During link rendering, Hugo checks if the link destination is a mounted source.
+If it's mounted, then Hugo checks if the current page is in the same project as the mount target.
+If it is in the same project, then Hugo replaces the link destination with the mount target.
+If it isn't mounted, and the mount target isn't in the same project, then Hugo uses the link as normal.
```

Comment:
I'm not too concerned about this being here, but it definitely falls into information I don't need as a person using these.

Reply:
I disagree. When I was working on this for Mimir/GEM it definitely helped me to think of it in terms of Hugo asking "Does this target URL also exist in this project? If yes, substitute for that topic."